src/background.ts

```diff
@@ -71,6 +71,16 @@ console.log(`Environment: ${import.meta.env.MODE}, appData: ${appName}`);
 // バージョン0.14より前の設定ファイルの保存場所
 const beforeUserDataDir = app.getPath("userData"); // マイグレーション用
 
+// app.getPath("userData")を呼ぶとディレクトリが作成されてしまうため空なら削除する。
+try {
+  fs.rmdirSync(beforeUserDataDir);
```

Comment:
おまじない程度ですが、再帰的消去の有無のデフォルト値を知らなくてもわかるようにしておくのはどうでしょう。
```suggestion
  fs.rmdirSync(beforeUserDataDir, {recursive: false});
```